daniel-planets.rb

```diff
@@ -0,0 +1,167 @@
+planets = [
+  {name: "Mercury",index_from_sun: 1,distance_from_sun: 0.4,
```

Comment:
A common format for an array of hashes to be written is:

```rb
[
  {
    key: "Value",
    key: "Value",
  }, {
    h2: "Value"
  }
]
```

Which does a good job to visually separate each hash without needing a whole line break.